renku/api/storage.py

```diff
@@ -88,14 +88,18 @@ def track_paths_in_storage(self, *paths):
 
     def pull_path(self, path):
         """Pull a path from LFS."""
-        client, commit, path = self.resolve_in_submodules(
-            self.repo.commit(), path
-        )
-        run(
-            self._CMD_STORAGE_PULL + [str(path)],
-            cwd=str(client.path.absolute()),
-            check=True
-        )
+        if self.use_external_storage and self.external_storage_installed:
+            client, commit, path = self.resolve_in_submodules(
+                self.repo.commit(), path
+            )
+            run(
+                self._CMD_STORAGE_PULL + [str(path)],
+                cwd=str(client.path.absolute()),
+                stdout=PIPE,
+                stderr=STDOUT,
+            )
+        elif self.use_external_storage:
+            raise errors.ExternalStorageNotInstalled(self.repo)
```

Comment:
We better readability we should consider documenting in docstring if a function raises something or not.